### DISCUSSION TITLE
ppocrlabel表格标注识别结果拖动问题

### DISCUSSION THREAD

**Discussion by PlayerJian** [PaddlePaddle/PaddleOCR#13399] [Q&A]
ppocrlabel标注工具，表格识别，识别结果在新增标注框时，需按顺序排序，拖动结果时识别结果列表可不可以滑动，或者拖动时列表自动滑动，要不然有几百的检测框，新增一个在最后面，要拖动到前面，非常复杂和耗费时间

**Comment by crackso** [comment]
1，这个问题应该反馈给ppocrlabel项目这边吧，目前这块应该是和主项目分离了；
2，工具问题，这个调整顺序确实很耗时，可以自己修改下，比如直接在框上调整编号，然后也可以使用快捷键直接编号等等，个人方法供参考；
![image](https://github.com/user-attachments/assets/94f8bc27-4f66-4f2b-b9cb-14d5268cb378)

![image](https://github.com/user-attachments/assets/2e16aa90-d68c-485f-9f91-4ff20f0b5917)

**Reply by lwinhong** [reply]
> 1，这个问题应该反馈给ppocrlabel项目这边吧，目前这块应该是和主项目分离了； 2，工具问题，这个调整顺序确实很耗时，可以自己修改下，比如直接在框上调整编号，然后也可以使用快捷键直接编号等等，个人方法供参考； ![image](https://private-user-images.githubusercontent.com/7133536/349059796-94f8bc27-4f66-4f2b-b9cb-14d5268cb378.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4NDkyNjUsIm5iZiI6MTcyMjg0ODk2NSwicGF0aCI6Ii83MTMzNTM2LzM0OTA1OTc5Ni05NGY4YmMyNy00ZjY2LTRmMmItYjljYi0xNGQ1MjY4Y2IzNzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MDVUMDkwOTI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTJlMzI0YzJiZTZjYTI3MmEwYzZjMWUxMTRhZWJiYmY3ZWIzYjNmZDI2ZTY0NTBhNzljOWJhZTA4ZDI3ZmM3MSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Y611nvjPPcpwgWrTUOI-kBV26hrnr3auVsKxTIpsrS4)
> 
> ![image](https://private-user-images.githubusercontent.com/7133536/349059083-2e16aa90-d68c-485f-9f91-4ff20f0b5917.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4NDkyNjUsIm5iZiI6MTcyMjg0ODk2NSwicGF0aCI6Ii83MTMzNTM2LzM0OTA1OTA4My0yZTE2YWE5MC1kNjhjLTQ4NWYtOWY5MS00ZmYyMGYwYjU5MTcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MDVUMDkwOTI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjNjZWMxNTUyZTNhZTY0YmVlOTVkMjk5Njk1ZDMzNTQ4NDc3N2JhOTY4NDFiZTRhZGI5YjUyNjkxN2M1ZWZmOCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.MZD_oE69RuDc-EGOc4ht3pswSa51XEUknh390aTdmHc)

我自己也加了这个功能，不然很崩溃啊
